front/src/common/IntervalsDataViz/style.scss

```diff
@@ -336,6 +336,36 @@ $resize-width: 3px;
               }
             }
           }
+          &.resizing-scale-x {
+            border-top: $scaling-height solid;
+            height: 50px;
+            width: 100%;
+            div {
+              height: 50px;
```

Comment:
you have some height in px and other in em, is it volontary ?

Reply:
Hum actually, I just copied-pasted the code above, so I don't really know

Reply:
But I think yes